### PR TITLE
Fixes #3226 - Update Code of Conduct contact points.

### LINKED ISSUE #3226: CoC points to an outdated eMail address
Ola is no longer working for Mozilla, so the eMail address in the Code of Conduct is no longer valid. Let's replace it with @ksy36's address, who volunteered. :)

### PR DESCRIPTION
**Pull request by denschub**
Unfortunately, Ola left the company, so the email address is dead. However, @ksy36 volunteered to be a future CoC contact point. 🎉 

r? @ksy36 
r? @miketaylr

### CODE REVIEW

**Comment by miketaylr** [comment]
@denschub can you file an issue and update the commit message / title? I think our changelog tooling will possibly explode on "No issue". :)

**Comment by denschub** [comment]
> I think our changelog tooling will possibly explode on "No issue".

Oh! Sure, I've updated the commit msg. :)

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abuse, harassment, or otherwise unacceptable behavior may be
-reported by contacting the project team at [Ola](mailto:ogasidlo@mozilla.com) / [@zoepage](https://github.com/zoepage) or [Dennis](mailto:DSchubert@mozilla.com) / [@denschub](https://github.com/denschub). All
+reported by contacting [Ksenia](mailto:kberezina@mozilla.com) or [Dennis](mailto:DSchubert@mozilla.com) / [@denschub](https://github.com/denschub). All
```
Comment (ksy36) [inline CODE_OF_CONDUCT.md]: Maybe we can use my github handle here too ( @ksy36 )

Comment (denschub) [reply CODE_OF_CONDUCT.md]: ✅

**Pull request merged**